README.md

```diff
@@ -33,6 +33,8 @@ The Mobility Data Specification (**MDS**), a project of the [Open Mobility Found
 
 * The [`geography`][geography] API endpoints are intended to be implemented by regulatory agencies and consumed by mobility providers. Providers query the Policy API to get information about geographical regions for regulatory and other purposes. It was first released in October 2019, originally included as part of the Policy specification. Development takes place under the guidance of the OMF's City Services Working Group.
 
+* **[Beta feature](/general-information.md#beta-features):** _(as of 1.1.0)_ The [`jurisdiction`][jurisdiction] API endpoints are intended to be implemented by regulatory agencies that have a need to coordinate with each other. The jurisdiction endpoints allow cities to communicate boundaries between one another and to mobility providers. It was first released in November 2020, originally included as part of the Jurisdiction specification. Development takes place under the guidance of the OMF's City Services Working Group.
```

Comment:
I don't think you need to call out the beta feature here on the home page. I think it can be done on the jurisdiction page.